air-quality-backend/shared/src/database/forecasts.py

```diff
@@ -80,3 +96,14 @@ def get_forecast_data_from_database(
     if location_name is not None:
         query["name"] = location_name
     return list(collection.find(query))
+
+
+def get_data_textures_from_database(

```

Comment:
Can we add some unit tests to cover this?

Reply:
unit tests added to `shared/tests/database/forecasts?test.py`